_sass/custom/nav.scss

```diff
@@ -12,7 +12,7 @@
 
   .side-bar-category-items.open {
     opacity: 1;
-    max-height: 550px;
+    max-height: 750px;
```

Comment:
We have enough content that we reached this max-height! This causes the Reference section to overlap the Guides list in the sidebar (🤮 ). I just increased this max-height a bit to fix this, but once we really have too much content to show, we'll need to spend a bit more time figuring out how to scroll the overflow.